{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/globals/navigation/mobile.css

```diff
@@ -162,17 +185,19 @@
 */
 .nav-Mobile_Dropdown,
 .nav-Mobile_DropdownChild {
-  position: absolute;
-  top: 0;
+  position: fixed;
```

Comment:
I think I see a potential to break, though I haven't tested it: if you have a third level of navigation that has a transform on the parent (e.g. you're transitioning it in) won't `position: fixed` create a new stacking context which would then make the third level positioned `var(--Mobile_NavHeight)` from the height of its parent and *not* from the document?

Reply:
Nice catch, tricky one to fix 🤔